src/utils/getAssetsUrl.test.ts

```diff
@@ -1,10 +1,36 @@
 import { ASSETS_URL } from './constants';
 import { getAssetsUrl } from './getAssetsUrl';
 
+const uploadCareImage = {
```

Comment:
Also, just to be safe, it's better to specify the type here, so that we're certain that the object structure is correct and up-to-date.